src/scss/_table.scss

```diff
@@ -0,0 +1,320 @@
+}
+
+.table-base tr {
+  padding: 0.35em;
+
+  border-bottom: 1px solid var(--color-bg-light);
+}
+
+.table-base tr:last-child {
+  border-bottom: none;
+}
+
+.table-base caption {
+  margin: 0.5em 0 0.75em;
+
+  font-size: 1.5em;
+
+  div {
+    display: flex;
+    justify-content: space-between;
+    padding-left: 1rem;
+  }
+}
+
+.table-base th {
+  padding: 0.625em;
+
+  font-weight: bold;
+
+  text-align: left;
+
+  white-space: nowrap;
+}
+
+.table-base td {
+  padding: 0.625em;
+
+  font-weight: lighter;
+}
+
+.table-base .-line-break {
+  word-wrap: break-word;
+}
+
+/* responsive style applied when screen width is reduced to < 600px */
+@media screen and (max-width: 600px) {
+  .table-base {
+    border: 0;
+  }
+
+  .table-base caption {
+    font-size: 1.3em;
+  }
+
+  .table-base thead {
+    position: absolute;
+
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    padding: 0;
+    overflow: hidden;
+
+    border: none;
+
+    clip: rect(0 0 0 0);
+  }
+
+  .table-base tr {
+    display: block;
+    margin-bottom: 0.625em;
+
+    border-bottom: 3px solid var(--color-lavender);
+  }
+
+  .table-base td {
+    display: block;
+
+    font-size: 0.8em;
+    text-align: right;
+
+    border-bottom: 1px solid var(--color-bg-light);
+  }
+
+  .table-base td::before {
+    float: left;
+
+    font-weight: bold;
+
+    content: attr(data-label);
+  }
+
+  .table-base td:last-child {
+    border-bottom: 0;
+  }
+}
+
+// end table-base
+
+// style below this should be removed when integrating new table with the rest of the application
+.table {
+  width: 100%;
+  margin: 1rem 0;
+  padding-bottom: 0.5rem;
+
+  font-size: 1rem;
+
+  background: var(--color-bg-dark);
+}
+
+.table .loading {
+  display: flex;
+
+  margin-top: 0.5em;
+  margin-left: 0.4em;
+}
+
+.table .row.preview {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.table .headers {
+  position: relative;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  margin-top: 0.3em;
+  padding: 0.3em 1rem;
+  padding-left: 1rem;
+
+  border-bottom: 1px solid var(--color-bg-light);
+}
+
+.table .header {
+  flex: 1;
+
+  font-size: 1.2rem;
+}
+
+.hooks.table .headers {
+  padding-left: 4.25rem;
+}
+
+.hook-status {
+  fill: none;
+}
+
+.hook-status.-success {
+  stroke: var(--color-green);
+}
+
+.hook-status.-failure {
+  stroke: var(--color-red);
+}
+
+.table .row .cell {
+  flex: 1;
+  align-items: center;
+  justify-content: center;
+  width: 300px;
+
+  font-weight: lighter;
+  text-align: center;
+  word-wrap: break-word;
+}
+
+.table .headers .header.-last {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 1rem;
+}
+
+.table .cell.-last {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 1rem;
+
+  button {
+    margin: 0.25rem 0;
+  }
+}
+
+.table .source-id {
+  min-width: 330px;
+}
+
+.table .headers .first-cell {
+  width: 2rem;
+}
+
+.filler-cell {
+  width: 6ch;
+}
+
+.table .headers .-label {
```

Comment:
is `.-label` attached to another class or is it a generic utility class?

Reply:
anything related to `.table` will be removed with hooks, consider these all dead